teams/vsp/teams/operations/daytime-oncall-support-role.md

```diff
@@ -54,13 +56,13 @@ Your Github team notifications and [#vsp-operations](https://dsva.slack.com/chan
 
 ### Support requests
 
-Create an issue describing the request in GitHub and link it to one of the two super-epics. Assign it to yourself. Track any work done with comments. Only track support that takes more than 15 mins and if it is a PR that takes under 15 mins please link to the super epic. If the support work seems to be taking you down a long road, make a note around this and bring to the team to scope out into a project. Unless an emergency doesn't spend more than 2 hours on any given support work. Summarize both failed attempts and final solutions so the issue can be referred to again in the future if necessary. If other team members are involved, add them to the ticket.
+If a support request takes more than 30 mins of your time, create a GitHub Issue describing the request and link it to one of the two Super Epics. Assign it to yourself. Track any work done with comments. If a support request takes less than 30 minutes and results in a PR, please link the PR to the super epic. If the support work seems to be taking you down a long road, make a note around this and bring to the team to scope out into a project. Unless it's an emergency, don't spend more than 2 hours on any given support work. Summarize both failed attempts and final solutions so the issue can be referred to again in the future if necessary. If other team members are involved, add them to the ticket.
```

Comment:
@jhouse-solvd I made some more changes to this paragraph to make it clearer. But... I'm not sure if the changes are accurate. Can you re-check this paragraph?

Reply:
This is good. I'm creating an issue to document support request guidance. That guidance can then be linked into this template. But, that's outside the scope of your pr. :)